oneanddone/users/templates/users/profile/detail.html

```diff
@@ -11,11 +11,16 @@
   <h1 id="user-profile-name">
     {{ _("<span>{name}</span>'s Profile")|fe(name=userprofile.name) }}
   </h1>
-{% if userprofile.personal_url %}
-  <h4 id ="user-profile-url">
- {{ _('Personal URL:') }} <a href="{{userprofile.personal_url }}" target="_blank"><span>{{ userprofile.personal_url }}</span></a>
- </h4>
-{% endif %}
+  {% if userprofile.personal_url %}
+    <h4 id ="user-profile-url">
+      {{ _('Personal URL:') }} <a href="{{userprofile.personal_url }}" target="_blank"><span>{{ userprofile.personal_url }}</span></a>
+    </h4>
+  {% endif %}
+  {% if userprofile.bugzilla_email %}
+    <h4 id ="bugzilla-email-id">
```

Comment:
You don't really need the `-id` in the value of the `id` of this element. It would make more sense to just use `id ="bugzilla-email"`